navigator/frontend/src/ui-core/src/LongIdentifier/index.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from "react";
+import { default as styled } from "../theme";
+
+const Identifier = styled.span`
+  cursor: pointer;
+  justify-content: space-between;
+  padding: 0.5em 1em;
+  align-items: center;
+  text-decoration: none;
+  margin-right: 1rem;
+  box-shadow: 0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2);
+  color: ${({ theme }) => theme.colorPrimary[1]};
+  &,
+  &:hover {
+    color: ${({ theme }) => theme.colorSecondary[1]};
+  }
+  border-radius: ${_ => "999rem"};
```

Comment:
Weird value for the border radius :D But as long as it works it doesn't matter :D

Reply:
I copied and pasted it from another similar style. I assume the intent was to make sure that the border was round on the sides no matter the size. I'm not a front-end engineer, so that's good enough for me. Thanks for raising this.